### PR TITLE
Finish mobile version

### PR DESCRIPTION
**Pull request by billymohajeri**
Hi

The next step to creating a portfolio website is to complete the design of the rest of the pages of the mobile version based on the given templates.
I tried to implement the rest of the content based on the template.

Thanks

### CODE REVIEW

**Review by OyePriscilla — CHANGES_REQUESTED** [review]
Hi @billymohajeri 😄  , 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

Kindly ensure that all your linters checks passed. I noticed that stylelint is not passing.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Comment by billymohajeri** [comment]
Hi

The next step to creating a portfolio website is to complete the design of the rest of the pages of the mobile version based on the given templates.
I tried to implement the rest of the content based on the template.
Sorry about stylelint errors. Now it's fixed.

Thanks

File: index.html [hunk 1]
```diff
@@ -20,43 +20,222 @@
...
+        <h3>Ruby on Rails</h3>
+        <h3>RSPec</h3>
+        <h3>CapyBara</h3>
+        <h3>Selenium</h3>
+        <h2>🟠 Skills</h2>
+        <h3>Codekit</h3>
+        <h3>Github</h3>
+        <h3>Codepen</h3>
+        <h3>Gitlab</h3>
+        <h3>Terminal</h3>
```
Comment (Favourezeugwa) [inline index.html:196]: - I think it is important that The technology categories ("Languages", "Frameworks" and "Skills") are in a `<ul>`, each of them in a `<li>`

- Also, the technology in each category(e.g., "JavasScript", "HTML" and "CSS") are also a list `<ul>` and each of them a `<li>`

You can simply re-factor your code to look something like this 👇🏽 
```
  <ul class="languages">
          <li class="frame">
            <h2>🔶Languages</h2>
          </li>
          <ul class="list">
            <li>javascript</li>
            <li>ruby</li>
            <li>html</li>
            <li>css</li>
          </ul>
     </ul>
    <ul class="Frameworks">
          <li>
            <h2>🟧 Frameworks</h2>
          </li>
          <ul class="list">
            <li>Bootstrap</li>
            <li>Ruby on Rails</li>
            <li>RSPec</li>
            <li>CapyBara</li>
            <li>Selenium</li>
          </ul>
      </ul>
```

- What do you think about this code snippet that I have added above?  You should include the rest of the list and can modify the class name as you wish but kindly remember to update your CSS file for the list tag as well. 🍒

File: styles.css [hunk 2]
```diff
@@ -90,6 +95,362 @@ ul {
...
+.about-me > h3 {
+  font-family: "Inter", sans-serif;
+  font-style: normal;
+  margin-bottom: 32px;
+  font-weight: 400;
+  font-size: 22px;
+  line-height: 43px;
+  text-align: center;
+  color: #fff;
+}
```
Comment (Favourezeugwa) [inline styles.css:342]: - Please kindly remember to update this part of your CSS to suite the technologies and categories `list tags` that you would have updated. 🌸 🌸

**Review by Favourezeugwa — CHANGES_REQUESTED** [review]
## STATUS: REQUIRES CHANGES 🚦 ♻️ 
Hi @billymohajeri , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

close lady

## Highlights 💯 
- No linter errors
- GitHub flow used

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Yazino12 — APPROVED** [review]
## Hi @billymohajeri 🖐🏾,

You've done well implementing the requested changes by a previous reviewer 🎉🎊🎉, there is nothing else left other than to merge it 🚀🚀🚀.

> You are following a good amount of coding best practices including DRY & CLEAN, HTML/CSS best practices, and this project is by far the best that I have reviewed in your cohort. Keep up the amazing work 🎉🍾🎊

### To highlight:
- Linters are passing ✔️
- Correct use of GitHub flow ✔️ 
- You documented your work professionally ✔
- Good commit messages ✔
- Good PR title and summary ✔

# STATUS:  APPROVED ✅✅✅
TRY AGAIN
_Check the comments under the review 👇🏽👇🏽👇🏽._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏 💻  

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please ping me @Yazino12 when you comment so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**